main.py

```diff
@@ -1,17 +1,51 @@
 import animate
 import pygame
 from pygame.locals import *
+from menu import *
```

Comment:
I think it's not good style to import everything via *. If we do, we lose track & oversight of module dependencies.
```suggestion
import menu
```